include/eve/module/core/regular/rotr.hpp

```diff
@@ -38,12 +38,64 @@ namespace eve
       constexpr int l = sizeof(element_type_t<T>) * 8;
       static_assert(eve::abs(S) < l, "[eve::rotr] Rotation is out of range.");
 
-      return EVE_DISPATCH_CALL(v,s);
+      return EVE_DISPATCH_CALL(v, s);
     }
 
     EVE_CALLABLE_OBJECT(rotr_t, rotr_);
   };
 
+  //================================================================================================
+  //! @addtogroup core_arithmetic
+  //! @{
+  //!   @var rotr
+  //!   @brief Bitwise rotation to the right
+  //!
+  //!   **Defined in Header**
+  //!
+  //!   @code
+  //!   #include <eve/module/core.hpp>
+  //!   @endcode
+  //!
+  //!   @groupheader{Callable Signatures}
+  //!
+  //!   @code
+  //!   namespace eve
+  //!   {
+  //!      template<unsigned_value T, integral_value N>
+  //!      T rotr(T x, N n) noexcept;
+  //!   }
+  //!   @endcode
+  //!
+  //!   **Parameters**
+  //!
+  //!     * `x`:  [argument](@ref eve::unsigned_value) to be rotated.
+  //!     * `n`:  [shift](@ref eve::integral_value).
+  //!
+  //!    **Return value**
+  //!
+  //!      The [elementwise](@ref glossary_elementwise) bitwise rotation to the right of the first
+  //!      parameter by the second.
+  //!
+  //!      The call `rotr(x, n)` is equivalent to `std::rotr(x, n)` if `x` is an [simd value](@ref
```

Comment:
same. we probably means to say : on scalar value OR to say it is done on each lanes.